game/src/sandbox/dashboards/travel_times.rs

```diff
@@ -483,19 +493,60 @@ fn contingency_table(ctx: &mut EventCtx, app: &App, filter: &Filter) -> Widget {
                             duration_buckets[idx + 1]
                         ),
                     }),
-                    Line(format!("Lost {} in total", total_loss)).fg(Color::hex("#EB3223")),
+                    Line(format!("Lost {} in total", total_loss.to_rounded_string(1)))
+                        .fg(Color::hex("#EB3223")),
                 ]),
             ));
         }
         x1 += bar_width;
     }
-    batch.extend(Color::BLACK, outlines);
+    // Draw the y-axis
+    let mut y_axis_ticks = GeomBatch::new();
+    let mut y_axis_labels = GeomBatch::new();
+    {
+        let line_length = 8.0;
+        let line_thickness = 2.0;
+
+        intervals.1[1..]
+            .iter()
+            .map(|interval| {
+                // positive ticks
+                let y =
+                    max_bar_height * (1.0 - interval.inner_seconds() / intervals.0.inner_seconds());
+                (interval, y)
+            })
+            .chain(
+                // negative ticks
+                intervals.1[1..].iter().map(|interval| {
+                    let y = total_height
+                        - max_bar_height
+                            * (1.0 - interval.abs().inner_seconds() / intervals.0.inner_seconds());
+                    (interval, y)
+                }),
+            )
+            .for_each(|(interval, y)| {
+                let start = Pt2D::new(0.0, y);
+                let line: geom::Line =
```

Comment:
The type declaration isn't necessary, but tiny nitpick

Reply:
I'll fix this in a followup commit, which I need anyway to do the whole `[rebuild] [release]` dance for github actions